NREL/wex
# Using libcurl built with Windows7

@sjanzou,

Using the windows 7-built Libcurl seems to be working for me on both Win32 and Win64 Debug mode. 

To test the build:

1.	Replace Libcurl folders in /wex
2.	See if openssl.exe works
3.	Delete the entire ssc /build_vs2017/Win32/Debug & /build_vs2017/x64/Debug folders
4.	Clean and rebuild wex and ssc
5.	See if Debug modes work

I checked if using the windows 10-built Libcurl would work if I deleted both build folders, but the problem is before that and is quickly reproducible: just opening openssl.exe gives the error.

Hoping this will work, and thanks!
